SNIClient.py

```diff
@@ -1087,14 +1088,125 @@ async def main():
             time.sleep(3)
             sys.exit()
         elif args.diff_file.endswith((".apbp", "apz3")):
-            adjustedromfile, adjusted = Utils.get_adjuster_settings(romfile, gui_enabled)
-            if adjusted:
-                try:
-                    shutil.move(adjustedromfile, romfile)
-                    adjustedromfile = romfile
-                except Exception as e:
-                    logging.exception(e)
-            asyncio.create_task(run_game(adjustedromfile if adjusted else romfile))
+            lastSettings = Utils.get_adjuster_settings("LttP")
```

Comment:
internal name for LttP is nowadays the full "A Link to the Past".

Reply:
I can change it to "A Link to the Past", that will not look good in persistent storage though
`last_settings_A Link to the Past: !!python/object:argparse.Namespace`
I could probably replace spaces with underscores in the function